### DISCUSSION TITLE
Usage qauestion./Feature request - Synchronized Tabs

### DISCUSSION THREAD

**Discussion by markxnelson** [thegeeklab/hugo-geekdoc#577] [Ideas]
Hi,

Is it possible to have multiple sets of tabs on a page and to synchronize them?  I want to have a page that has several code examples, and each code example would be provided in different languages, e.g. Java, JavaScript, Python.  If a user chooses the "Java" tab in any of the tab groups, I would like all of the tab groups to swtich to the Java tab.

I believe the Learn theme has a feature like this with grouped tabs.  They have a groupId, and all tabs with same groupId change when you change any one.

If this is not currently possible in geekdoc,, I would like to request it please :) Please let me know the right way to do that, if opening an issue like this is not the right way.

Thanks

**Comment by xoxys** [comment]
Hi, thanks for your request. Sadly, that's not possible with this theme. The Learn theme is using JavaScript for the Tab synchronization, while we use a pure CSS implementation. As we try to avoid JavaScript as much as possible with this theme and only implement JavaScript features that can be easily disabled without affecting the usability of the theme, I don't think we will implement this feature, sorry.

If you still want to have this feature, Hugo allows adding custom shortcodes to your deployment regardless of the used theme.
